<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>488</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Define Tool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Tool Number</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_2"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Tool Type</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="toolTypeCB">
                <item>
                 <property name="text">
                  <string>None Selected...</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Straight</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Tapered</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Conical</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Ball</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Tapered Ball</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Tool Name</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="imageLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QStackedWidget" name="stackedWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="currentIndex">
               <number>2</number>
              </property>
              <widget class="QWidget" name="blankPage"/>
              <widget class="QWidget" name="straightPage">
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <layout class="QFormLayout" name="straightToolForm">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_11">
                    <property name="text">
                     <string>Diameter</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit_8"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_12">
                    <property name="text">
                     <string>Cut Length</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineEdit_9"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_13">
                    <property name="text">
                     <string>Tool Length</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="lineEdit_10"/>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_30">
                    <property name="text">
                     <string>Shaft Diameter</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLineEdit" name="lineEdit_28"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="taperedPage">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <layout class="QFormLayout" name="taperedToolForm">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_14">
                    <property name="text">
                     <string>Top Diameter</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit_11"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_15">
                    <property name="text">
                     <string>Bottom Diameter</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineEdit_12"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_16">
                    <property name="text">
                     <string>Cut Length</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="lineEdit_13"/>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_17">
                    <property name="text">
                     <string>Tool Length</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLineEdit" name="lineEdit_14"/>
                  </item>
                  <item row="4" column="0">
                   <widget class="QLabel" name="label_29">
                    <property name="text">
                     <string>Shaft Diameter</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="1">
                   <widget class="QLineEdit" name="lineEdit_27"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="conicalPage">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <layout class="QFormLayout" name="conicalToolForm">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_18">
                    <property name="text">
                     <string>Top Diameter</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit_15"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_19">
                    <property name="text">
                     <string>Cut Angle</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineEdit_16"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_20">
                    <property name="text">
                     <string>Tool Length</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="lineEdit_17"/>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_28">
                    <property name="text">
                     <string>Shaft Diameter</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLineEdit" name="lineEdit_26"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="ballPage">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <layout class="QFormLayout" name="ballToolForm">
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit_18"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_22">
                    <property name="text">
                     <string>Tool Length</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineEdit_19"/>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_21">
                    <property name="text">
                     <string>Ball Diameter</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_27">
                    <property name="text">
                     <string>Shaft Diameter</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="lineEdit_25"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="taperedBallPage">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <layout class="QFormLayout" name="formLayout_3">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>Top Diameter</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit_20"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_23">
                    <property name="text">
                     <string>Ball Diameter</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineEdit_21"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_24">
                    <property name="text">
                     <string>Cut Length</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="lineEdit_22"/>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_25">
                    <property name="text">
                     <string>Tool Length</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLineEdit" name="lineEdit_23"/>
                  </item>
                  <item row="4" column="0">
                   <widget class="QLabel" name="label_26">
                    <property name="text">
                     <string>Shaft Diameter</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="1">
                   <widget class="QLineEdit" name="lineEdit_24"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_2"/>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Speeds and Feeds (recommended)</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="2" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Plunge Feed Rate</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEdit_4"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Spindle Speed (rpm)</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="lineEdit_5"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>% Step Over</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="lineEdit_6"/>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Depth of Cut</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLineEdit" name="lineEdit_7"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Feed Rate</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_3"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
